src/k8s/deployment.ts

```diff
@@ -2,13 +2,28 @@ import { Progress } from "../util/progress";
 import * as Odo from "../odo";
 import { QuickPickItem } from "vscode";
 import * as common from './common';
+import { ClusterExplorerV1 } from 'vscode-kubernetes-tools-api';
 
-export namespace Command {
-   export function deploy(build: string) {
+export class DeploymentConfigNodeContributor implements ClusterExplorerV1.NodeContributor {
+    contributesChildren(parent: ClusterExplorerV1.ClusterExplorerNode | undefined): boolean {
+        return !!parent && parent.nodeType === 'resource' && parent.resourceKind.manifestKind === 'DeploymentConfig';
+    }
+
+    async getChildren(parent: ClusterExplorerV1.ClusterExplorerNode | undefined): Promise<ClusterExplorerV1.Node[]> {
+        return common.getNode(Command.getAllReplicas(parent), 'Deploy', 'deploy');
+    }
+}
+
+export class Command {
+    static getAllReplicas(parent: ClusterExplorerV1.ClusterExplorerNode) {
```

Comment:
Function should be named 'getReplicationControllers' to match 'getDeploymentConfig' function name.